hack/test-cmd.sh

```diff
@@ -337,12 +330,19 @@ for test in "${tests[@]}"; do
   os::cmd::try_until_text "oc get projects -o name" "project/${namespace}"
   os::test::junit::declare_suite_end
 
-  ${test}
+  if ! ${test}; then
+    failed="true"
+    tail -40 "${LOG_DIR}/openshift.log"
```

Comment:
Do we want to keep the big honking "[FAIL] !!!!! Test Failed !!!!", or is that superfluous?